cabot/launch/cabot2-gtm-outdoor-remote.launch

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) 2020  Carnegie Mellon University
+ 
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+ 
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+ 
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE. -->
+
+<launch>
+  <!--
+      @output
+      value can be 'log' or 'screen' to switch standard output
+  -->
+  <arg name="output" default="log"/>
+
+  <node pkg="nodelet" type="nodelet" output="$(arg output)" name="cabot_nodelet_manager"
+	args="manager"/>
+  
+  <!--
+      Motor Controller Adapter
+      Convert cmd_vel (linear, rotate) speed to motor target (left, right) speed.
+  -->
+  <node pkg="nodelet" type="nodelet" output="$(arg output)" name="odriver_adapter_node"
+	args="load MotorAdapter/ODriverNodelet cabot_nodelet_manager">
+    <param name="encoder_topic" type="string" value="/motorStatus" />
+    <param name="odom_topic" type="string" value="/odom_raw" />
+    
+    <param name="motor_topic" type="string" value="/motorTarget" />
+    <param name="cmd_vel_topic" type="string" value="/cmd_vel" />
+    
+    <param name="max_acc" value="1.2"/>
+    <param name="target_rate" value="20"/>
+    <param name="bias" value="0.26"/> <!--wheel separation-->
+    
+    <param name="gain_omega" value="1.0"/>
+    <param name="gain_omega_i" value="0.0"/>
+    <remap from="/imu" to="/cabot/imu/data"/>
+  </node>
+  
+  <!--
+      Motor Controller (ODrive)
+  -->
+  <node pkg="odriver" type="odriver_node.py" name="odriver_node" output="log">
+    <param name="wheel_diameter" value="0.200" />
+    <param name="count_per_round" value="90" />
+    <param name="left_is_1" value="false" />
+    <param name="is_clockwise" value="false" />
```

Comment:
Are the motors for outdoor model anti-clockwise rotation as default?

Reply:
Yes, they are anti-clockwise rotation as default for some reason.